corehq/apps/hqwebapp/models.py

```diff
@@ -51,3 +53,8 @@ def get_latest_alert(cls):
             return active_alerts[0]
         else:
             return ''
+
+
+class ApiKeySettings(models.Model):
+    api_key = models.OneToOneField(ApiKey, on_delete=models.CASCADE)
+    ip_whitelist = ArrayField(models.GenericIPAddressField(), default=list, null=True, blank=True)
```

Comment:
I don't think this needs `null=True` since it defaults to a blank list. Similarly, not sure `blank=True` is contributing anything.